src/ui/public/partials/paginated_selectable_list.html

```diff
@@ -38,12 +42,28 @@
     ng-click="sortHits(hits)"
     aria-live="assertive"
   >
-    <span class="euiScreenReaderOnly">Sort by</span>
-    Name
+    <span class="euiScreenReaderOnly"
+      i18n-id="common.ui.paginateSelectableList.sortByButtonLabelScreenReaderOnly"
+      i18n-default-message="Sort by"
+    ></span>
+    <span class="euiScreenReaderOnly"
```

Comment:
That's wrong. That class was only set for "Sort by", but not for the "Name" because that should still be visible.